kitsune/questions/views.py

```diff
@@ -187,9 +187,15 @@ def question_list(request, product_slug):
             question_qs = question_qs.done()
 
     question_qs = question_qs.select_related("creator", "last_answer", "last_answer__creator")
-    question_qs = question_qs.prefetch_related("topic", "topic__product")
+    # Exclude questions over 90 days old without an answer or
+    # older than 5 years or
+    # created by deactivated users
+    today = date.today()
+    question_qs = question_qs.exclude(
+        created__lt=today - timedelta(days=90), num_answers=0, creator__is_active=False
+    ).filter(updated__gt=today - timedelta(days=365 * 5))
```

Comment:
I'd be inclined to leave this at 1 year, for now, or a lesser increase like 2 years - we can always bump it up further in future if performance is really good.